### DISCUSSION TITLE
How do i disable all styles for one sveltekit layout?

### DISCUSSION THREAD

**Discussion by FarhanAliRaza** [skeletonlabs/skeleton#1319] [Q&A]
I have a problem. When i try to create a new layout all styles automatically apply to it. I want use custom css for that layout. But i can not.

**Comment by Sarenor** [comment]
As said in https://github.com/skeletonlabs/skeleton/issues/1320 only open one channel of communication please.

**Reply by FarhanAliRaza** [reply]
Sorry for opening two different channels.
![Screenshot (31)](https://user-images.githubusercontent.com/62690310/231727506-634c0548-1fdf-4440-9e74-1b8634d64c61.png)
My Dashboard Layout with styles
![Screenshot (30)](https://user-images.githubusercontent.com/62690310/231727591-3d809634-3863-41d6-a0c7-d006a4286024.png)
My new layout without styles

I do not have any root layout .. As you can see in both images

**Reply by Sarenor** [reply]
That is unexpected.

I'd have expected the Skeleton-CSS to only affect the one layout path and am not quite sure how it affects other parts of the site, we'll definitely have to investigate this.

Might have been better to do this in the bug instead of the discussion, although I'm not 100% if it's a Skeleton or a Svelte/SvelteKit issue
